### DISCUSSION TITLE
[QUESTION] nestjs-query only with typeorm

### DISCUSSION THREAD

**Discussion by LeoKaynan** [doug-martin/nestjs-query#1559] [General]
Can I use the library without nestjs? only with the ORM.

**Comment by smolinari** [comment]
It's built for running with NestJS only. Why don't you want to use NestJS?

Scott

**Reply by LeoKaynan** [reply]
NestJS can sometimes generate invalid dependencies.

In my case, I would like to use Apollo in version 4 with fastify, but it is not yet supported by Nestjs, so I would not use nestjs but I would like to have generated my queries with filters and pagination.

**Reply by smolinari** [reply]
If you want to use Fastify, you could also use Mercurius with Nest. IMHO, Mercurius is a better more performant package too. 

Scott

**Reply by LeoKaynan** [reply]
Thanks. I'll take a look.
